client/blocks/reader-post-card/featured-video.jsx

```diff
@@ -15,7 +15,7 @@ import FeaturedImage from './featured-image';
 class FeaturedVideo extends React.Component {
 
 	static propTypes = {
-		thumbnailUrl: React.PropTypes.string,
+		thumbnailUrlPromise: React.PropTypes.object,
```

Comment:
There are a few bugs in the code I posted above. The code is there merely to illustrate, not to copy and paste :wink: